cartridges/app_custom_account_address/cartridge/controllers/Account.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+/**
+ * @namespace Account
+ */
+
+var server = require("server");
+
+var csrfProtection = require("*/cartridge/scripts/middleware/csrf");
+server.extend(module.superModule);
+
+/**
+ * Account-SubmitRegistration : The Account-SubmitRegistration endpoint is the endpoint that gets hit when a shopper submits their registration for a new account
+ * @name Base/Account-SubmitRegistration
+ * @function
+ * @memberof Account
+ * @param {middleware} - server.middleware.https
+ * @param {middleware} - csrfProtection.validateAjaxRequest
+ * @param {querystringparameter} - rurl - redirect url. The value of this is a number. This number then gets mapped to an endpoint set up in oAuthRenentryRedirectEndpoints.js
+ * @param {httpparameter} - dwfrm_profile_customer_firstname - Input field for the shoppers's first name
+ * @param {httpparameter} - dwfrm_profile_customer_lastname - Input field for the shopper's last name
+ * @param {httpparameter} - dwfrm_profile_customer_phone - Input field for the shopper's phone number
+ * @param {httpparameter} - dwfrm_profile_customer_email - Input field for the shopper's email address
+ * @param {httpparameter} - dwfrm_profile_customer_emailconfirm - Input field for the shopper's email address
+ * @param {httpparameter} - dwfrm_profile_login_password - Input field for the shopper's password
+ * @param {httpparameter} - dwfrm_profile_login_passwordconfirm: - Input field for the shopper's password to confirm
+ * @param {httpparameter} - dwfrm_profile_customer_addtoemaillist - Checkbox for whether or not a shopper wants to be added to the mailing list
+ * @param {httpparameter} - csrf_token - hidden input field CSRF token
+ * @param {category} - sensitive
+ * @param {returns} - json
+ * @param {serverfunction} - post
+ */
+server.replace(
+    'SubmitRegistration',
```

Comment:
Why do we need this replace ?

Reply:
Here in the form i had to submit data only for email and password.